WalletWasabi.Documentation/RPC.md

```diff
@@ -0,0 +1,426 @@
+# Wasabi Remote Procedure Call interface
+
+Wasabi wallet provides an RPC interface to interact with the wallet programatically. The RPC server is listing by default on port 18099.
+
+## Limitations
+
+The RPC server does NOT support batch requests nor TLS communication (because it is not supported on Linux nor Mac: https://github.com/dotnet/corefx/issues/14691). Requests are served in order one by one in serie (no parallel processing).
```

Comment:
```suggestion
The RPC server does NOT support neither batch requests nor TLS communications (because they are not supported on Linux and on Mac: https://github.com/dotnet/corefx/issues/14691). Requests are served in order one by one in series (no parallel processing).
```

Reply:
No, in this case TLS is not supported. The request batching is just not implemented.

Reply:
Check again please, I updated that.